Index: extras/Products.ATContentTypes/Products/ATContentTypes/tests/atcttestcase.py
===================================================================
--- extras/Products.ATContentTypes/Products/ATContentTypes/tests/atcttestcase.py	(revision 112076)
+++ extras/Products.ATContentTypes/Products/ATContentTypes/tests/atcttestcase.py	(working copy)
@@ -31,4 +31,6 @@
 from Products.PloneTestCase.setup import portal_owner
 ZopeTestCase.installProduct('SiteAccess')
+
+from plone.app.imaging import testing
 PloneTestCase.setupPloneSite(extension_profiles=['Products.CMFPlone:testfixture'])
 
@@ -68,8 +70,10 @@
 
 class ATCTSiteTestCase(PloneTestCase.PloneTestCase):
-    pass
+
+    layer = testing.imaging
 
 class ATCTFunctionalSiteTestCase(PloneTestCase.FunctionalTestCase):
-    pass
+
+    layer = testing.imaging
 
 class ATCTTypeTestCase(ATCTSiteTestCase):
@@ -79,4 +83,6 @@
     """
 
+    layer = testing.imaging
+
     klass = None
     cmf_klass = None
