Index: extras/plone.app.linkintegrity/plone/app/linkintegrity/tests/layer.py
===================================================================
--- extras/plone.app.linkintegrity/plone/app/linkintegrity/tests/layer.py	(revision 34440)
+++ extras/plone.app.linkintegrity/plone/app/linkintegrity/tests/layer.py	(working copy)
@@ -2,4 +2,5 @@
 from Products.Five import zcml, fiveconfigure
 from collective.testcaselayer.ptc import BasePTCLayer, ptc_layer
+from plone.app.imaging import testing
 from StringIO import StringIO
 from base64 import decodestring
@@ -54,3 +55,3 @@
 
 
-integrity = PloneLinkintegrity(bases=[ptc_layer])
+integrity = PloneLinkintegrity(bases=[testing.imaging, ptc_layer])
